backends/events/sdl/sdl-events.cpp

```diff
@@ -63,6 +70,31 @@ static uint32 convUTF8ToUTF32(const char *src) {
 
 	return utf32;
 }
+
+void SdlEventSource::loadGameControllerMappingFile() {
+	bool loaded = false;
+	if (ConfMan.hasKey("gcdbfile")) {
+		Common::FSNode file = Common::FSNode(ConfMan.get("gcdbfile"));
+		if (file.exists()) {
+			if (SDL_GameControllerAddMappingsFromFile(file.getPath().c_str()) < 0)
+				error("File %s not valid: %s", file.getPath().c_str(), SDL_GetError());	
+			else {
+				loaded = true;
+				debug("Game controller DB file loaded: %s", file.getPath().c_str());
+			}
+		}
+	}
+	if (!loaded && ConfMan.hasKey("extrapath")) {
```

Comment:
I'm not too fond of using `extrapath` directly. Usually ScummVM looks up files using the SearchManager which also includes platform specific paths. However SearchMan cannot give the full path for the files (as they are not necessarily directly on the filesystem). That being said I have no idea how to improve this while keeping the code simple. I guess this is fine and can always be improved later if needed.